File: VB/WindowsApplication8/AppMenuFileLabelExtension.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraEditors

Namespace WindowsApplication8
	Public Module AppMenuFileLabelExtension

        <System.Runtime.CompilerServices.Extension()> _
        Public Sub AddSeparator(ByVal fileLabel As AppMenuFileLabel)
            Dim l As New LabelControl()
            l.AutoSizeMode = LabelAutoSizeMode.None
            l.Text = String.Empty
            l.Height = 10
            l.LineOrientation = LabelLineOrientation.Horizontal
            l.LineVisible = True
            fileLabel.Parent.Controls.Add(l)
            l.Dock = DockStyle.Top
        End Sub
	End Module
End Namespace

File: CS/WindowsApplication8/AppMenuFileLabelExtension.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraEditors;

namespace WindowsApplication8
{
    public static class AppMenuFileLabelExtension
    {
        
        public static void AddSeparator(this AppMenuFileLabel fileLabel)
        {
            LabelControl l = new LabelControl();
            l.AutoSizeMode = LabelAutoSizeMode.None;
            l.Text = string.Empty;
            l.Height = 10;
            l.LineOrientation = LabelLineOrientation.Horizontal;
            l.LineVisible = true;
            fileLabel.Parent.Controls.Add(l);
            l.Dock = DockStyle.Top;
        }
    }
}
